quinn/src/connection.rs

```diff
@@ -883,6 +958,11 @@ pub enum WriteError {
     /// Unknown stream
     #[error(display = "unknown stream")]
     UnknownStream,
+    /// This was a 0-RTT stream and the server rejected it.
+    ///
+    /// Can only occur on clients, for streams initiated before the handshake completes.
```

Comment:
Should we have consistent wording on this comment and the `ReadError` variant?

Reply:
Yup, fiddled with the wording a bit more and made it consistent.